.gitignore

```diff
@@ -25,3 +25,5 @@ $RECYCLE.BIN/
 Thumbs.db
 UserInterfaceState.xcuserstate
 .env
+./vscode-data.json
```

Comment:
this is very useful for the autocmpletion in vscode, you don't want to ignore it ;)